values.yaml

```
@@ -1196,6 +1196,9 @@ store_gateway:
     metricRelabelings: []
     # -- Additional endpoint configuration https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#endpoint
     extraEndpointSpec: {}
+  statefulSet:
+    # https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies
```

```suggestion
    # -- https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies
```

check helm-docs for the specific format https://github.com/norwoodj/helm-docs